core/src/main/java/hudson/security/BasicAuthenticationFilter.java

```diff
@@ -98,7 +98,6 @@ public void init(FilterConfig filterConfig) throws ServletException {
     }
 
     @Override
-    @SuppressWarnings("ACL.impersonate")
```

Comment:
`ACL.impersonate` seems unlikely to be a warning. Seems the author thought that this was a justification rather than a value.